CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
 cmake_minimum_required(VERSION 2.6)
 
 project(libbarrett)
-
-set(libbarrett_VERSION_MAJOR 1)
-set(libbarrett_VERSION_MINOR 3)
 set(libbarrett_VERSION_MAJOR 1)
 set(libbarrett_VERSION_MINOR 2)
-set(libbarrett_VERSION_PATCH 3)
+set(libbarrett_VERSION_PATCH 4)
```

Comment:
We need to re-make the doxy files (html) and post them at web.barrett.com/libbarrett.